ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/ManageSubscriptionsView.kt

```diff
@@ -191,29 +214,83 @@ private fun NoActiveSubscriptionsViewPreview() {
     ManageSubscriptionsView(
         screen = noActiveScreen,
         localization = testData.localization,
+        support = testData.support,
         purchaseInformation = null,
-        onPathButtonPress = {},
+        onAction = {},
     )
 }
 
 @OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
 @Composable
 private fun ManageSubscriptionsButtonsView(
     screen: CustomerCenterConfigData.Screen,
-    onDetermineFlow: (CustomerCenterConfigData.HelpPath) -> Unit,
+    onButtonPress: (CustomerCenterConfigData.HelpPath) -> Unit,
     useOutlinedButton: Boolean = false,
 ) {
     Column {
         screen.supportedPaths.forEach { path ->
             ManageSubscriptionButton(
                 path = path,
-                onDetermineFlow = onDetermineFlow,
+                onDetermineFlow = onButtonPress,
                 useOutlinedButton = useOutlinedButton,
             )
         }
     }
 }
 
+@OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+@Composable
+private fun OtherPlatformSubscriptionButtonsView(
+    localization: CustomerCenterConfigData.Localization,
+    support: CustomerCenterConfigData.Support,
+    managementURL: Uri?,
+    onAction: (CustomerCenterAction) -> Unit,
+    useOutlinedButton: Boolean = false,
+) {
+    val buttonModifier = Modifier
+        .fillMaxWidth()
+        .padding(vertical = 4.dp)
+
+    Column {
+        managementURL?.let {
+            Button(
+                onClick = { onAction(CustomerCenterAction.OpenURL(it)) },
+                modifier = buttonModifier,
+                colors = ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.primary),
+            ) {
+                Text(
+                    text = localization.commonLocalizedString(
+                        CustomerCenterConfigData.Localization.CommonLocalizedString.MANAGE_SUBSCRIPTION,
+                    ),
+                    color = if (useOutlinedButton) {
+                        MaterialTheme.colorScheme.primary
+                    } else {
+                        MaterialTheme.colorScheme.onPrimary
+                    },
+                )
+            }
+        }
+        support.email?.let {
+            Button(
```

Comment:
I believe much of this is repeated... Should we extract it to another composable?

Reply:
oh I can actually reuse the other button. I think when I started the PR I couldn't reuse, but now I can

Reply:
oh right, actually I need another one because these have hardcoded values. So I will extract another `Composable`